lib/types/cql-utils.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+const types = require("./index");
+const rust = require("../../index");
+const { ResponseError } = require("../errors");
+const TimeUuid = require("./time-uuid");
+const Uuid = require("./uuid");
+const Duration = require("./duration");
+
+/**
+ * Wrap value into QueryParameterWrapper based on the type
+ * @param {CqlType} type
+ * @param {*} value
+ * @returns {rust.QueryParameterWrapper}
+ */
+function getWrapped(type, value) {
+    if (value === null || value === types.unset) {
+        return null;
+    }
+    // To increase clarity of the error messages, if we call some function on value,
+    // type is checked explicitly. In other cases default Error will be thrown.
```

Comment:
:question: :wrench: It's not clear to me what scenario this describes.

Reply:
:question:  What function? What value?

Reply:
I hope the comment is more meaningful now.